exonum-java-binding-common/src/test/java/com/exonum/binding/common/serialization/StandardSerializersRoundtripTest.java

```diff
@@ -6,12 +6,12 @@
 import com.exonum.binding.common.hash.HashCode;
 import com.google.common.collect.ImmutableList;
 import java.util.List;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
-public class StandardSerializersRoundtripTest {
+class StandardSerializersRoundtripTest {
 
   @Test
-  public void roundtripLongTest() {
+  void roundtripLongTest() {
```

Comment:
These are very good candidates for a parameterized test (they actually were ones in exonum-serialization project that used JUnit 5 from the start):

```java
  @ParameterizedTest
  @ValueSource(longs = {Long.MIN_VALUE, -1L, 0L, 1L, Long.MAX_VALUE})
  void roundtripLongTest(Long v) {
    roundTripTest(v, StandardSerializers.longs());
  }
```